lib/puppet/application/filebucket.rb

```diff
@@ -142,6 +169,21 @@ def backup
     end
   end
 
+  def list
+    #Default fromdate : Epoch
+    fromdate = options[:fromdate] || "0:0:0 1-1-1970"
```

Comment:
It seems like we do this exact defaulting in dipper.rb and file.rb.  Is it necessary to do the defaulting here when we could just pass nil through the indirector for these options?